### ISSUE TITLE
Far cry 5, heavy freezes, could be a DXVK bug or wine bug...

### ISSUE THREAD

**Issue by gabriele2000** [doitsujin/dxvk#2265]
I filed a [bug](https://bugs.winehq.org/show_bug.cgi?id=51697) on wine bugzilla, just to be sure, ntdll was spamming like hell but I also get a lot of `D3D11Buffer::QueryInterface: Unknown interface query` and I guess it's DXVK telling me something that I don't understand, yet.

Basically while I play I get a lot of freezes, a lot of random lag and since it's the only game (for now) that does this, I decided to file a bug.

[Video of proof](https://mega.nz/file/F58HCKwb#twzX6cTEP7wDTSSbM_5UkUHMn5jwLY67j1w6q-JXvbE)

### System information
- GPU: Nvidia (ugh) GTX 1050TI 4GB (craptop version)
- Driver: 470.57.2
- Wine version: 6.16-staging-tgk
- DXVK version: [this one](https://github.com/doitsujin/dxvk/commit/2786bbda6498a1dad7e372d75744532a17f10048)

### Apitrace file(s)
- Not yet

### Log files
- d3d9.log: nope
- d3d11.log: [FarCry5_d3d11.log](https://github.com/doitsujin/dxvk/files/7071672/FarCry5_d3d11.log)
- dxgi.log: [FarCry5_dxgi.log](https://github.com/doitsujin/dxvk/files/7071666/FarCry5_dxgi.log)

**Comment by doitsujin** [comment]
just looks like the shader stutter that has been happening in every single game for the past three and a half years?

**Comment by gabriele2000** [comment]
WINEDEBUG=-ntdll doesn't fix the performance, as I explained in the other thread and `D3D11Buffer::QueryInterface: Unknown interface query` spam is even more evident

**Comment by gabriele2000** [comment]
> just looks like the shader stutter that has been happening in every single game for the past three and a half years? I don't really see any issue here.

No, it happens everytime, that game just doesn't have infinite shader creation, it's obviously not a shader problem, since in every other game it lags for the first time, not everytime.

I'll provide a tracelog, just for you to see that probably I'm right

**Comment by doitsujin** [comment]
I still don't really see how this is a bug at all, if the `QueryInterface` spam causes perf issues then just disable DXVK logging I guess.

**Comment by gabriele2000** [comment]
Yeah I'll disable HD texture pack, that's the issue, no apitrace will be useful since this is not actually a DXVK problem from what I gathered...

By the way, why do I have the QueryInterface spam? What does it mean?

**Comment by doitsujin** [comment]
It means that the game is doing something stupid that's not supposed to work, but isn't actually an issue.

**Comment by aufkrawall** [comment]
> EDIT2: Vram is not full with them on

It doesn't have to be "full" (report maximum usage) to run out of VRAM if it fails to allocate large chunks. 4GB is by far not enough even without DXVK when using the texture pack and DXVK can't manage VRAM pressure like the Nvidia D3D11 driver can do (not even AMD D3D11 can [might not come as a surprise]).

**Comment by gabriele2000** [comment]
> > EDIT2: Vram is not full with them on
> 
> It doesn't have to be "full" (report maximum usage) to run out of VRAM if it fails to allocate large chunks. 4GB is by far not enough even without DXVK when using the texture pack and DXVK can't manage VRAM pressure like the Nvidia D3D11 driver can do (not even AMD D3D11 can [might not come as a surprise]).

Well, that's the only game that has this problem... all thanks to ubisoft, thanks for that "hd" texture pack that is poorly made, so poorly made that I'm not the only one that has this problem, even windows users with an nvidia 1080ti

**Comment by gabriele2000** [comment]
Closed because it's not a DXVK bug
